be/src/column/column.h

```diff
@@ -110,6 +110,13 @@ class Column {
 
     virtual void resize(size_t n) = 0;
 
+    // If the column has already overflow, upgrade to one larger Column type,
```

Comment:
Why  upgrade when the column already overflow?
Why not upgrade before it overflow?

Reply:
When building big chunks, I only need to do one upgrade after building finish

Reply:
Doing so has no effect on other current processes， Otherwise, when AppendColumn, you have to interpret what type of BinaryColumn it is.

Reply:
This itself is not a frequent behavior, there is no need to introduce too much extra judgment and code complexity for it

Reply:
One HashJoin, at most  upgrade once